src/api/routes/articlesRouter.js

```diff
@@ -56,6 +57,6 @@ articlesRouter.delete('/comments/:commentId', verifyToken, checkParameter, delet
 articlesRouter.get('/:slug/comments', getComment);
 articlesRouter.post('/:slug/rating', verifyToken, validateBody('validateRating'), slugExist, createRatings);
 articlesRouter.put('/:slug/rating', verifyToken, validateBody('validateRating'), slugExist, UpdateRatings);
-
+articlesRouter.post('/report/:slug', verifyToken, validateBody('checkComment'), slugExist, reportArticle);
```

Comment:
Please consider the order of this route order `/api/articles/:slug/report` I think it should look better and easier when we'll be on front-end side.